src/fides/api/models/privacy_experience.py

```diff
@@ -48,7 +48,7 @@ class ComponentType(Enum):
 FidesJSOverlayTypes: List[ComponentType] = [
     ComponentType.banner_and_modal,
     ComponentType.modal,
-    ComponentType.tcf_overlay
+    ComponentType.tcf_overlay,
```

Comment:
My local `black` formatter is probably configured differently than it's supposed to be